assets/src/dashboard/animations/parts/simpleAnimation.js

```diff
@@ -38,24 +44,37 @@ function SimpleAnimation(animationName, keyframes, timings) {
       if (!target.current) {
         return () => {};
       }
-      const effect = new KeyframeEffect(target.current, keyframes, timings);
+      const effect = createKeyframeEffect(target.current, keyframes, timings);
       return hoistAnimation(new Animation(effect, document.timeline));
     }, [hoistAnimation]);
 
-    return <FullSizeAbsolute ref={target}>{children}</FullSizeAbsolute>;
+    return useClippingContainer ? (
+      <FullSizeAbsolute style={{ overflow: 'hidden' }}>
+        <FullSizeAbsolute ref={target}>{children}</FullSizeAbsolute>
+      </FullSizeAbsolute>
+    ) : (
+      <FullSizeAbsolute ref={target}>{children}</FullSizeAbsolute>
+    );
```

Comment:
The `zoom` animation requires a clipping container so we can zoom in and not just have the item get bigger and bigger.